_includes/staff-pagination-links.html

```diff
@@ -0,0 +1,160 @@
+{% for p in pagination.pages %}
```

Comment:
for this page, i noticed that the pagination file didn't work properly with _exactly_ 5 pages of content. I made some fixes here that probably just need to be ported back to the original pagination file rather than added as a second type